demo/admin/src/documents/pages/blocks/FullWidthImageBlock.tsx

```diff
@@ -1,32 +0,0 @@
-import { messages } from "@comet/admin";
-import { createCompositeBlock, createOptionalBlock } from "@comet/blocks-admin";
-import { DamImageBlock } from "@comet/cms-admin";
-import { customBlockCategory } from "@src/common/blocks/customBlockCategories";
-import { RichTextBlock } from "@src/common/blocks/RichTextBlock";
-import { FormattedMessage } from "react-intl";
-
-const FullWidthImageContentBlock = createOptionalBlock(RichTextBlock, {
-    title: <FormattedMessage {...messages.content} />,
-});
-
-export const FullWidthImageBlock = createCompositeBlock(
-    {
-        name: "FullWidthImage",
-        displayName: <FormattedMessage id="blocks.fullWidthImage" defaultMessage="Full Width Image" />,
-        blocks: {
-            image: {
-                block: DamImageBlock,
-                title: <FormattedMessage {...messages.image} />,
-                paper: true,
-            },
-            content: {
-                block: FullWidthImageContentBlock,
-                title: <FormattedMessage {...messages.content} />,
-            },
-        },
-    },
-    (block) => {
-        block.category = customBlockCategory;
```

Comment:
We should keep an example for custom block categories somewhere

Reply:
Readded 651cd3874c55df9b0133a616edbebe8da747ef05